nym-wallet/src-tauri/capabilities/main-capability.json

```diff
@@ -0,0 +1,31 @@
+{
+    "$schema": "../gen/schemas/desktop-schema.json",
+    "identifier": "main-capability",
+    "description": "Default capability for Nym Wallet main window",
+    "windows": [
+        "main",
+        "nymWalletApp",
+        "log"
+    ],
+    "platforms": [
+        "linux",
+        "macOS",
+        "windows"
+    ],
+    "permissions": [
+        "core:default",
+        "core:path:default",
+        "core:event:default",
+        "core:window:default",
+        "core:app:default",
+        "core:resources:default",
+        "core:menu:default",
+        "core:tray:default",
+        "shell:allow-open",
```

Comment:
Suggestion: for opener it is encouraged to switch on https://tauri.app/plugin/opener/ (and drop shell)

Reply:
Dropped this too